src/plugins/application_config/.i18nrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "prefix": "applicationConfig",
+  "paths": {
+    "applicationConfig": "."
+  },
+  "translations": ["translations/ja-JP.json"]
+}
```

Comment:
Same as above. Can we use central file - https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.i18nrc.json?